pkg/services/librarypanels/librarypanels_test.go

```diff
@@ -15,78 +16,251 @@ import (
 )
 
 func TestCreateLibraryPanel(t *testing.T) {
-	libraryPanelScenario(t, "When an admin tries to create a library panel", "then it should fail if library panel already exists", func(t *testing.T) {
-		lps, context := setupTestEnv(t, models.ROLE_ADMIN, map[string]string{})
-		command := getCreateCommand(1, "Text - Library Panel")
+	t.Run("When an admin tries to create a library panel that already exists", func(t *testing.T) {
```

Comment:
Why don't you use the `libraryPanelScenario` abstraction any longer, to define test cases? I think it makes sense, and is more ergonomic than the new pattern here? 
```suggestion
	t.Run("When an admin tries to create a library panel that already exists, it should fail", func(t *testing.T) {
```

Reply:
The simple answer is that I couldn't debug the individual t.Run within the libraryPanelScenario, but as you mentioned that I shouldn't have t.Run within t.Run I'll see what I can come up with.

Reply:
Fixed